src/js/components/Provenance/DatasetProvenance.tsx

```diff
@@ -38,31 +38,35 @@ const DatasetProvenance = ({ metadata, loading }: DatasetProvenanceProps) => {
 
         {/* --- CREATOR, PRIVACY, LICENSES, KEYWORD ---*/}
         {metadata.privacy || metadata.licenses.length || metadata.keywords.length ? (
-          <Descriptions style={{ paddingTop: '20px' }}>
+          <div style={{ display: 'flex', flexDirection: 'column', paddingTop: '20px' }}>
```

Comment:
is it possible to have this stay as one descriptions component like we talked about yesterday, or was there another reason that wouldn't work?

Reply:
only problem is forgetting to commit changes....